src/domTree.js

```diff
@@ -339,13 +401,23 @@ class symbolNode {
 /**
  * SVG nodes are used to render stretchy wide elements.
  */
-class svgNode {
-    constructor(children, attributes) {
+class svgNode implements VirtualDomNode {
+    children: SvgChildNode[];
+    attributes: Attribute[];
```

Comment:
In other places we provide `attributes` as `attributes: {[string]: string};` we should do the same here.  Not sure how much work is involved with that.  If it looks like a lot feel free to add a TODO and tackle it later.

Reply:
Done.